assets/app/views/_templateopt.html

```diff
@@ -29,6 +30,21 @@
           autocapitalize="off"
           spellcheck="false"
           ng-attr-aria-describedby="{{parameter.description ? (paramID + '-description') : undefined}}">
+      <textarea
+          ng-if="parameter.multiline"
+          ng-attr-id="{{paramID}}"
+          ng-attr-name="{{paramID}}"
+          class="form-control"
+          rows="4"
+          placeholder="{{ parameter | parameterPlaceholder }}"
+          ng-model="parameter.value"
+          ng-required="parameter.required && !parameter.generate"
+          ng-blur="cleared = false"
+          ng-trim="false"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
```

Comment:
Can't decide if we should remove the autocorrect / autocapitalize / spellcheck off attributes. If it's a textarea, it's often regular text. If it's a cert, you'd paste it in anyway?